javalib/src/main/scala/java/util/Properties.scala

```diff
@@ -0,0 +1,64 @@
+package java.util
+
+import java.{util => ju}
+
+import scala.collection.JavaConversions
+import scala.collection.JavaConversions._
+
+class Properties(protected val defaults: Properties)
+    extends ju.Hashtable[AnyRef, AnyRef] {
+
+  def this() = this(null)
+
+  def setProperty(key: String, value: String): AnyRef =
+    put(key, value)
+
+  // def load(reader: Reader): Unit
+  // def load(inStream: InputStream): Unit
+  // @deprecated("", "") def save(out: OutputStream, comments: String): Unit
+  // def store(writer: Writer, comments: String): Unit
+  // def store(out: OutputStream, comments: String): Unit
+  // def loadFromXML(in: InputStream): Unit
+  // def storeToXML(os: OutputStream, comment: String): Unit
+  // def storeToXML(os: OutputStream, comment: String, encoding: String): Unit
+
+  def getProperty(key: String): String = {
+    get(key) match {
+      case value: String => value
+
+      case _ =>
+        if (defaults != null) defaults.getProperty(key)
+        else null
+    }
```

Comment:
I think it would make more sense to implement `getProperty(value)` in terms of `getProperty(value, defaultValue = null)`, and have the latter call `defaults.getProperty(value, defaultValue)`.